### PR TITLE
Floodlights

### PR DESCRIPTION
**Pull request by musicmanvr**
WE HATE STRUCTURE YAML!!!
## About the PR

Added tall, small, and LZ floodlights for more long-lasting and less eye-shredding sources of light.

MOTE: 
## Why / Balance

I'm going to make flares less harsh in a future pr, but floodlights at the LZ and floodlights around the FOB allow for better ways to light an area.

Both floodlights are destructible, repairable, but the small one will be destroyed and drop materials as apposed to the large one requiring upkeep.

Both give off infinite sources of light for the moment until power is implemented in a proper way.
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

https://github.com/user-attachments/assets/cf5f14ba-1646-4c76-97dd-1e3f0e5868f5

To repair the larger lights, there's a note to inspect for any marine to see.
Unscrew, crowbar, weld, add 2 cable, then screw shut to fix.
## Breaking changes

**Changelog**

:cl:
- add: Added floodlights to help be a light in the darkness.

### CODE REVIEW

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
YML formats are
Parent
ID
Name
Description

The rest seems ok but we'll check it just in case after

**Comment by musicmanvr** [comment]
> YML formats are Parent ID Name Description
> 
> The rest seems ok but we'll check it just in case after

i think i got all of em'

File: Resources/Prototypes/_RMC14/Catalog/Cargo/requisitions_catalog.yml [hunk 1]
```diff
@@ -122,6 +122,8 @@
 #        crate: CMCrateInflatableBarriers
       - cost: 2000
         crate: CMCrateReplacementLights
+      - cost: 2500
+        crate: RMCCrateFloodlights
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Catalog/Cargo/requisitions_catalog.yml:126]: There's no crate of floodlights that I can see in 13.
image

```suggestion
```

File: Resources/Prototypes/_RMC14/Catalog/Fills/Crates/engineering.yml [hunk 2]
```diff
@@ -173,3 +173,13 @@
...
+
+- type: entity
+  id: RMCCrateFloodlights
+  parent: CMCrateConstruction
+  name: floodlight crate (x4)
+  components:
+  - type: StorageFill
+    contents:
+      - id: RMCSmallFloodlight
+        amount: 4
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Catalog/Fills/Crates/engineering.yml:185]: Same as above
```suggestion
```

File: Resources/Prototypes/_RMC14/Entities/Objects/Misc/floodlights.yml [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    netsync: false
+    energy: 5
+    offset: "0, 0"
+    color: "#ECFFFF"
+    mask: /Textures/_RMC14/Effects/LightMasks/quad_cone.png
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 150
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Objects/Misc/floodlights.yml:32]: I don't think these are destructible in 13 but I'd have to check

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: Resources/Prototypes/_RMC14/Recipes/Construction/Graphs/structures/floodlightbig.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - to: completed
+      steps:
+      - tool: Screwing
+        doAfter: 2
+
+  - node: completed
+    entity: RMCTallFloodlight
+
+
+
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Recipes/Construction/Graphs/structures/floodlightbig.yml:51]:
```suggestion

```

File: Resources/Prototypes/_RMC14/Entities/Objects/Misc/floodlights.yml (hunk 3 above)
Comment (musicmanvr) [reply Resources/Prototypes/_RMC14/Entities/Objects/Misc/floodlights.yml:32]: I wanted to make sure they're not used as indestructible barricades

Comment (DrSmugleaf) [reply Resources/Prototypes/_RMC14/Entities/Objects/Misc/floodlights.yml:32]: Makes sense

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**